recipe/meta.yaml

```diff
@@ -1,39 +1,44 @@
-{% set version = "1.3.2" %}
+{% set name = "blosc" %}
```

Comment:
Why the name change? Shouldn't this still be `python-blosc`? Note we do package [`blosc`]( https://github.com/conda-forge/blosc-feedstock ) also.